bin/make-base-vm

```diff
@@ -153,6 +159,32 @@ fi
 # Remove cron to work around vmbuilder issue when umounting /dev on target
 removepkg=cron
 
+if [ $DOCKER = "1" ]; then
+
+  addpkg=`echo $addpkg | tr ',' ' '`
+
+  mkdir -p docker
+  cd docker
+
+  # Generate the dockerfile
+  cat << EOF > $OUT.Dockerfile
+FROM $DISTRO:$SUITE
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get --no-install-recommends -y install $addpkg
+
+RUN useradd -ms /bin/bash -U $DISTRO
+USER $DISTRO:$DISTRO
+WORKDIR /home/$DISTRO
+
+CMD ["/bin/bash"]
```

Comment:
I assume this is why there is the `-t` flag.  If you want it to just stay up, why not explicitly have a wait+sleep loop here?  It seems cleaner.

Reply:
I don't think we can do that. The line is supposed to be of the form `CMD [“executable”, “param1”, “param2”…]`. I don't know how such a loop would be written in that form.

Reply:
`CMD ["bash", "-c", "while true; do sleep 1; done"]` should work

Reply:
Actually, you can do `CMD ["sleep", "infinity"]`

Reply:
Done